chargeio/services/Transactions/TransactionService.cs

```diff
@@ -110,7 +112,7 @@ public virtual Transaction GetTransaction(string transactionId)
 
         public virtual SearchResults<Transaction> Transactions(
             int page = 1, 
-            int page_size = 20, 
+            int pageSize = 20, 

```

Comment:
Yes. The pageSize is only used for the query and has no impact on serialization.